homeassistant/components/monzo/strings.json

```diff
@@ -4,6 +4,10 @@
       "pick_implementation": {
         "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
       },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "The Monzo integration needs to re-authenticate your account"
+      },
```

Comment:
Isn't this already in there?

Reply:
I don't think so?

Reply:
I mean, by default.

Now that I say this I check withings and we also have it there. Lovely.